lib/plugin/instance.rb

```diff
@@ -250,9 +250,9 @@ def add_preloaded_topic_list_custom_field(field)
   end
 
   # Add a permitted_create_param to Post, respecting if the plugin is enabled
-  def add_permitted_post_create_param(name)
+  def add_permitted_post_create_param(name, type = "")
```

Comment:
This API is a bit odd to me. You pass an object of the type you want?

I would prefer an enum of something or a symbol.

Reply:
@eviltrout I made the change to use symbols. The internals look a lot better, but the api is less consistent now with how strong params are handled in controllers.